cinema/models.py

```diff
@@ -101,27 +142,30 @@ def validate_ticket(row, seat, cinema_hall, error_to_raise):
                     }
                 )
 
-    def clean(self):
+    def clean(self) -> None:
         Ticket.validate_ticket(
             self.row,
             self.seat,
             self.movie_session.cinema_hall,
-            ValidationError,
+            ValidationError
```

Comment:
Why did you remove this?) It is a nit, but generally a good practice to leave commas in the last row
So that when you add next row, github won't highlight it as changed